app/routes/pages.js

```diff
@@ -10,10 +10,16 @@ module.exports = function() {
   router.get('/settings.json', settings.main);
 
   router.get('/', pages.main);
-  router.get(/embed\/(.*)/, function(req, res, next) {
+
+  router.get(/embed\/(treemap|piechart|barchart|linechart|bubbletree|table|map)\/(.*)/, function(req, res, next) {
```

Comment:
One of the tasks was to get the view names dynamically from the babbage.ui package so that we do not have to hardcode it here, nor handle it each time we add new views to babbage.ui, so this implementation is not very flexible.